src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Color {
+    Blue,
```

Comment:
You should fix (or answer to) all mentor's comments before re-request new review. Be attentive.
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/01181ef8-e6a4-41ed-986b-00625818984c)